content/tutorials/portenta-h7/por-ard-ap/content.md

```diff
@@ -0,0 +1,131 @@
+
+![PortentaAccessPoint shown on the list of available network devices](assets/por_ard_ap_find_ap.png?sanitize=true)
+
+Now open a browser window on your mobile device and copy & paste the URL containing Portenta’s IP address that is displayed on the serial monitor. 
+
+![*The URL containing the IP address of the access point displayed in the serial monitor* ](assets/por_ard_ap_copy_ip_address.png?sanitize=true)
+
+Once you’ve entered the URL, the client sends a GET request to the web server to fetch the HTML web page specified in the code. Once loaded you will see the web page in your mobile browser. 
+
+![The HTML web page accessed on your mobile browser](assets/por_ard_ap_access_webpage.png?sanitize=true)
+
+## 6. Access the Board From your Mobile Device
+
+If you take a look at the serial monitor, you can see the details of the HTTP GET request and other details of the device connected to the access point. The GET request is always in the following format: 
+
+```cpp
+GET URL HTTP/1.1
+```
+
+The URL is a string of characters sent to the server, in this case /Hx (where x stands for the color of the LED). This request containing the URL is received on the server and it replies with the following HTTP/1.1 response indicating that the connection was successful:
+
+```cpp
+HTTP/1.1 200 OK
+```
+
+Once the server has responded to this request, it closes the connection and continues listening to next GET requests. 
+
+![The client details displayed on the serial monitor](assets/por_ard_ap_client_details.png?sanitize=true)
+
+You’re now be able to toggle the states of the red, green and blue LED through the buttons displayed on your mobile browser. Everytime you press a button, the client sends a GET request to a URL in the format /Hx or /Lx ,where x can be ‘r’, ‘g’ or ‘b’, depending on the button pressed on the HTML page. The web server then reads the URL requested by the client, changes the state of the LED corresponding to the URL and closes the connection. 
+
+![The GET request details displayed in the serial monitor](assets/por_ard_ap_toggle_LEDS.png?sanitize=true)
+
+# Conclusion
+
+This tutorial shows one of the several capabilities of the on-board WiFi+Bluetooth module by configuring the board as an access point and setting up a web server. You have also learnt how a simple client-server model and the underlying HTTP requests and responses work. 
+
+# Next Steps 
+
+Now that you've learnt how to set up a board as an access point and understand the client-server model, start experimenting with the **simpleWebServer.ino** sketch. This sketch can be tweaked in a variety of ways based on your needs. For example, you can add a slider to the HTML page that changes the blink rate of the built-in RGB LED from your mobile device.
+
+**Author :** Lenard George
```

Comment:
Check the formatting in the other documents. Use a single line break but add a space at the end of the line to avoid empty lines.